DevelopedCalculator.js

```diff
@@ -20,10 +20,12 @@ class DevelopedCalculator extends Calculator {
         super.onButtonNumberClick(event);
         // pushing the history array with current result
         this.history.push(this.currentResult);
+        // calling gettingHistoryArray
```

Comment:
It wil lbe good to add in the comments the reason of calling `this.gettingHistoryArray`